### DISCUSSION TITLE
35. SOP 정책이 필요한 이유와 관련된 공격 문제를 함께 설명해주세요.

### DISCUSSION THREAD

**Discussion by linirini** [jewelry-box/cs-interview#36] [General]
.

**Comment by jongmee** [comment]
대표적으로 CSRF 공격이 있다. Cross Site Request Forgery의 약어로 사용자의 동의 없이 다른 웹 사이트에 명령을 보내는 공격이다. `<img src="https://www.example.com/index.php?action=delete&id=123" />`와 같이 다른 곳으로 이동하는 링크에 공격자가 원하는 요청을 넣는다.
